spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteDataFilesSparkAction.java

```diff
@@ -175,11 +179,21 @@ public RewriteDataFiles.Result execute() {
 
     Stream<RewriteFileGroup> groupStream = toGroupStream(ctx, fileGroupsByPartition);
 
+    Builder resultBuilder;
```

Comment:
Nit: how about a method so we can still assign resultBuilder in one line?

``` Builder resultBuilder = doExecute(ctx, groupStream, ...)```

I guess we will have to rename doExecute .  doBatchExecute?

Reply:
sure, I can rename and inline this